### DISCUSSION TITLE
Install flux v2 from helm chart

### DISCUSSION THREAD

**Discussion by petrusvr** [fluxcd/flux2#2319] [Q&A] [answered]
Hi,
is there possibility to install flux on cluster from helm chart. I think I have seen this possibility for flux v1.

Consider below situation. I have a **MainCluster**. This **MainCluster** is used to manage other clusters from CI/CD.
1. **MainClaster** is bootstraped with flux from CLI and synching state with GitLab repo.
2. **MainClaster** also have crossplane.io used to provision AWS **EKSClusters**.
3. Whenever **MainCluster** provision new **EKSCluster** I want to bootstrap this **EKSCluster** with flux and configure to synch with GitLab repo

So, in step 3 it looks like using HelmRelease to deploy and setup flux is best aproach. Or maybe I'm missing something.
Can someone give me advice how to solve this problem differently than implementing my own Docker witch will bootstrap EKSCluster with flux?
Thx

**Comment by jack-evans** [comment, marked as answer]
https://github.com/fluxcd-community/helm-charts is probably what you are looking for

**Reply by nicogl** [reply]
@jack-evans Is there any way to bootstrap Flux with the Helm Chart? Not just installing its controllers and components.

**Reply by hlascelles** [reply]
@nicogl Did you find an answer to that?

**Reply by stefanprodan** [reply]
Bootstrap is a Flux CLI only feature, Helm has no such capabilities. Helm knows how to apply YAMLs, it doesn't create Git repos and setup deploy keys.

**Reply by hlascelles** [reply]
OK cool :+1:

I have installed the helm chart, and added manifests that add the secrets and GitrRepository. The source controller authenticates and syncs the Git repo correctly, all that works great... But then, no more actions. The repo contains a few simple k8s Resources but Flux does not reconcile and "make it so". It's as though it gets the repo, but then does nothing.

I was hoping/wondering if `bootstrap` did one more thing like "actually start reconciling".

`flux get sources all -A`shows the GitRepo "Ready" and "Suspended (false)". 

Nothing in the logs shows errors or that it is "seeing any files". None have been `excluded`. Will keep digging. Any other thoughts appreciated!

**Reply by stefanprodan** [reply]
Please see the get started guide https://fluxcd.io/flux/get-started/

**Comment by flickerfly** [comment]
You can use `flux install --export` to get the manifests to install similar to how you would with Helm, but just flat manifests if that is your intent.
